python/lsst/pipe/tasks/setPrimaryFlags.py

```diff
@@ -92,67 +256,28 @@ def run(self, sources, skyMap=None, tractInfo=None, patchInfo=None,
             Tract object
         patchInfo : `lsst.skymap.PatchInfo`
             Patch object
-        includeDeblend : `bool`
-            Include deblend information in isPrimary?
         """
-        nChildKey = None
-        if includeDeblend:
-            nChildKey = self.schema.find(self.config.nChildKeyName).key
-
-        # coadd case
+        # Mark the sources inside of a tract/patch
```

Comment:
"Mark whether sources are contained within the inner regions of the given tract/patch and are not "pseudo" (e.g. sky) sources."